echoshader/new_version/echogram.py

```diff
@@ -17,35 +29,77 @@ def __init__(self, MVBS_ds):
             "Image": {
                 "cmap": "jet",
                 "colorbar": True,
-                "cmap": "jet",
                 "tools": ["box_select", "lasso_select", "hover"],
                 "invert_yaxis": False,
                 "width": 600,
-            }
+            },
+            "RGB": {"tools": ["hover"], "invert_yaxis": False, "width": 600},
         }
 
-    def echogram(
+        self._init_widget()
+
+    def _init_widget(self):
+        """
+        Initialize the interactive widgets for controlling the echogram visualization.
+        """
+        self.channel_select = panel.widgets.Select(
+            name="Channel Select", options=self.MVBS_ds.channel.values.tolist()
+        )
+
+        self.Sv_range_slider = panel.widgets.EditableRangeSlider(
+            name="Sv Range Slider",
+            start=self.MVBS_ds.Sv.actual_range[0],
```

Comment:
I think we can do the overall change from `MVBS_ds` to just `ds`. 

To ensure people put in the right type of dataset, I think we should add a input checking method that is called in `Echogram.__init__` to check for the following:
- existence of the `Sv` data variable
- `Sv` data variable should have dimensions ("channel", "ping_time", "echo_range")

We should also allow `depth` in the last dimension, but we can do that later -- I'll create an issue so that we do not forget.

Reply:
Hey @leewujung, issue for this has been created: https://github.com/OSOceanAcoustics/echoshader/issues/98
 please add more details if you want.